Rover/config.h

```diff
@@ -66,6 +66,11 @@
 # define MODE_FOLLOW_ENABLED AP_FOLLOW_ENABLED
 #endif
 
+//////////////////////////////////////////////////////////////////////////////
+// Angle Sensor Driver - useful for wind vanes
+#ifndef ANGLESENSOR_ENABLED
+  #define ANGLESENSOR_ENABLED ENABLED
```

Comment:
Does this work? One problem with moving it to AP vehicle is that you can't see these vehicle level config files anymore. We typically have to get round it with dummy methods, which is annoying. `AP_AIS` would be a example.